From: Andrey Konovalov <andrey.konovalov@linaro.org>
Date: Fri, 28 Apr 2017 21:37:37 +0300
Subject: [PATCH] OCDM patch for chromium-65.0.3315.0.r527534.igalia.1

---

diff --git a/chrome/renderer/media/chrome_key_systems_provider.cc b/chrome/renderer/media/chrome_key_systems_provider.cc
index e2a90b7..45f3c07 100644
--- a/chrome/renderer/media/chrome_key_systems_provider.cc
+++ b/chrome/renderer/media/chrome_key_systems_provider.cc
@@ -7,6 +7,8 @@
 #include "base/time/default_tick_clock.h"
 #include "chrome/renderer/media/chrome_key_systems.h"
 
+#include "media/cdm/ppapi/external_open_cdm/src/browser/opencdm_key_systems.h"
+
 #include "widevine_cdm_version.h"  // In SHARED_INTERMEDIATE_DIR.
 
 ChromeKeySystemsProvider::ChromeKeySystemsProvider()
@@ -25,6 +27,7 @@ void ChromeKeySystemsProvider::AddSupportedKeySystems(
     test_provider_.Run(key_systems);
   } else {
     AddChromeKeySystems(key_systems);
+    AddExternalOpenCdmKeySystems(key_systems);
   }
 
   has_updated_ = true;
diff --git a/chrome/renderer/BUILD.gn b/chrome/renderer/BUILD.gn
index cffc929..dbb1c4e 100644
--- a/chrome/renderer/BUILD.gn
+++ b/chrome/renderer/BUILD.gn
@@ -138,6 +138,7 @@ static_library("renderer") {
     "//media",
     "//media:media_features",
     "//media/capture",
+    "//media/cdm/ppapi/external_open_cdm/src/browser:renderer",
     "//mojo/public/cpp/bindings",
     "//net",
     "//ppapi/features",
-- 
2.7.4
