From 5cac8936bd4573ecaab1b8523369d2b8ae74088c Mon Sep 17 00:00:00 2001
From: Andrey Konovalov <andrey.konovalov@linaro.org>
Date: Fri, 28 Apr 2017 21:37:37 +0300
Subject: [PATCH] OCDM patch for chromium-53.0.2785.143

---

diff --git a/chrome/renderer/media/chrome_key_systems_provider.cc b/chrome/renderer/media/chrome_key_systems_provider.cc
index e2a90b7..45f3c07 100644
--- a/chrome/renderer/media/chrome_key_systems_provider.cc
+++ b/chrome/renderer/media/chrome_key_systems_provider.cc
@@ -7,6 +7,8 @@
 #include "base/time/default_tick_clock.h"
 #include "chrome/renderer/media/chrome_key_systems.h"
 
+#include "media/cdm/ppapi/external_open_cdm/src/browser/opencdm_key_systems.h"
+
 #include "widevine_cdm_version.h"  // In SHARED_INTERMEDIATE_DIR.
 
 ChromeKeySystemsProvider::ChromeKeySystemsProvider()
@@ -25,6 +27,7 @@ void ChromeKeySystemsProvider::AddSupportedKeySystems(
     test_provider_.Run(key_systems);
   } else {
     AddChromeKeySystems(key_systems);
+    AddExternalOpenCdmKeySystems(key_systems);
   }
 
   has_updated_ = true;
-- 
2.7.4

